src/app/pages/CreateConnectorPage/StepNamespace.tsx

```diff
@@ -104,6 +107,15 @@ const ClustersGallery: FunctionComponent = () => {
     onRefresh();
   };
 
+  const getEvalNamespaceAlert = (expiration: string): string => {
+    const { hours, min } = getPendingTime(new Date(expiration));
+    console.log(hours, min);
```

Comment:
It'd be good to comment this out or remove unless you expect this to be troublesome.

Reply:
yes, this was not intentional I will remove it

Reply:
done

Reply:
no worries, I forget them all the time :-)